protos/telemetry/telemetry.proto

```diff
@@ -286,3 +295,47 @@ enum LandedState{
     LANDED_STATE_TAKING_OFF = 3;
     LANDED_STATE_LANDING = 4;
 }
+
+// Odometry message type.
+message Odometry {
+    // Mavlink frame id
+    enum MavFrame {
+        UNDEF = 0;
+        MAV_FRAME_BODY_NED = 8; // Setpoint in body NED frame. This makes sense if all position control is externalized - e.g. useful to command 2 m/s^2 acceleration to the right.
```

Comment:
I wonder if we should expose the `MAV_FRAME` name or just use `BODY_NED`. @JonasVautherin opinions?

Reply:
You're right in that `BODY_NED` should be understandable. It's following the same naming as `PositionBody` etc.

`BODY_NED` is good for me :+1:.